raise-all-market-products.sh

```diff
@@ -35,15 +36,19 @@ collectRepos() {
     .name'
 }
 
+showMigratedRepos() {
+  if [ -f "${workDir}/migrated-repos.txt" ]; then
```

Comment:
minor duplication: I'd assign the file to a variable and re-use it
log="${workDir}/migrated-repos.txt"